napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
As the commit message says: IBMDriver disabled because documentation and requirements are missing.

Because the requirements are not complete importing napalm fails.